src/rdo.rs

```diff
@@ -355,9 +355,48 @@ fn compute_tx_distortion<T: Pixel>(
   distortion
 }
 
-pub fn compute_rd_cost<T: Pixel>(fi: &FrameInvariants<T>, rate: u32, distortion: u64) -> f64 {
+pub fn compute_rd_cost_biased_by_importance<T: Pixel>(
```

Comment:
You only ever use this version, right? Why not just call it compute_rd_cost?

Reply:
So if I remove the original `compute_rd_cost`, call this function `compute_rd_cost`, then what should I call `compute_rd_cost_biased`?

Reply:
compute_rd_cost_biased could just be merged in, right? After all you're only ever using the biased version.

Reply:
Yeah. I suppose I'll just do that.